src/request.ts

```diff
@@ -140,6 +140,18 @@ export const getUser = async <A>(): Promise<{id: string, username: string, email
   return { id, username, email }
 }
 
+export const getTeam = async <A>(): Promise<{id: string, name: string}> => {
+  let daemonResponse
+  try {
+    daemonResponse = await axios.get(baseUrl() + '/team')
+    
+  } catch (err) {
+    throw processAxiosError(err)
+  }
+  const { id, name } = daemonResponse.data.value
```

Comment:
Move this 2 to lines 147 and 148, and also line 144 just add it to line 145 and make it a const, the reason is that you are handling the error by reformatting and rethrowing and so if an error happens lines 151 and lines 152 are never executed, to that point then you don't need those lines there, you can simply handle everything in the same try block, and also makes it easier to read/understand the code.